main.py

```diff
@@ -65,33 +85,67 @@ def do_create_room(self, arg):
         Dojo().create_room(arg)
 
     @docopt_cmd
-    def do_add_person(self,arg):
+    def do_add_person(self, arg):
         """Usage: add_person <person_fname> <person_lname> <FELLOW/STAFF> [<wants_accommodation>]"""
         check = Dojo().add_person(arg)
-        if check == True:
-                Dojo().assign_room(arg)
+        if check:
+            Dojo().allocate_room(arg)
```

Comment:
Hello @njirap,
When i changed to `Dojo.allocate_room(arg)` i get an error:

![screen shot 2017-03-08 at 5 59 26 pm](https://cloud.githubusercontent.com/assets/25608335/23714682/6e6d1868-042a-11e7-875e-a90186f72467.png)

Is it better if i probably allocate the instance of Dojo() to a variable? Something like:

`
self.dojo = Dojo()
self.dojo.allocate_room(arg)
`